plugins/com.google.cloud.tools.eclipse.appengine.deploy/src/com/google/cloud/tools/eclipse/appengine/deploy/AppEngineStandardDeployCommandHandler.java

```diff
@@ -0,0 +1,24 @@
+package com.google.cloud.tools.eclipse.appengine.deploy;
+
+import org.eclipse.core.commands.AbstractHandler;
+import org.eclipse.core.commands.ExecutionEvent;
+import org.eclipse.core.commands.ExecutionException;
+import org.eclipse.jface.dialogs.MessageDialog;
+import org.eclipse.ui.handlers.HandlerUtil;
+
+public class AppEngineStandardDeployCommandHandler extends AbstractHandler {
+
+  @Override
+  public Object execute(ExecutionEvent event) throws ExecutionException {
+    new MessageDialog(HandlerUtil.getActiveShell(event), "Deploying to App Engine Standard", null,
```

Comment:
Done, although this dialog will almost certainly not make it to the final code, but the code to use externalized strings are going to be useful later.